packages/agoric-cli/tools/getting-started.js

```diff
@@ -14,13 +14,18 @@ import { makePspawn } from '../src/helpers.js';
 const RETRY_BLOCKHEIGHT_SECONDS = 3;
 const SOCKET_TIMEOUT_SECONDS = 2;
 
+// TODO: Set this to `true` when `agoric install $DISTTAG` properly updates the
+// getting-started workflow's dependencies to current `@endo/*` and Agoric SDK
+// from the local registry.
```

Comment:
seems like this should reference an issue.

I don't see one that's quite on target; closest I find is #8621

Reply:
I'll try to add to that issue or create a new one to include as a reference in the TODO.